lang/gcc10-bootstrap/Portfile

```diff
@@ -69,13 +69,13 @@ checksums           gcc-10.3.0.tar.gz \
 extract {
     foreach f ${distfiles} {
         set f [lindex [split ${f} :] 0]
-        ui_msg "Extracting: ${f}"
+        ui_debug "Extracting: ${f}"
         if {[string match "*bz2*" ${f}]} {
-            ui_msg "${workpath} \"/usr/bin/tar xjf ${distpath}/${f}\""
+            ui_debug "${workpath} \"/usr/bin/tar xjf ${distpath}/${f}\""
             system -W ${workpath} "/usr/bin/tar xjf ${distpath}/${f}"
```

Comment:
`system` already [logs what it does](https://github.com/macports/macports-base/commit/26b5ff60905efcbc36bc1d2574b01479612e5ce3); you shouldn't need a separate `ui_debug` to log it a second time.

Reply:
good point, thanks